code/modules/mapping/dmm_suite.dm

```diff
@@ -1,4 +1,4 @@
-dmm_suite{
+/datum/maploader{

```

Comment:
If you're changing this file, throw away {}'s and relative pathing.

Reply:
do this as a code cleanup in a later PR or the diff will be unreadable